.github/workflows/test.yaml

```diff
@@ -8,9 +8,40 @@ on:
     branches:
     - 'main'
 
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
```

Comment:
This is a cool new feature I found in the docs. It automatically cancels existing runs when new commits are pushed to a PR.

Reply:
There used to be a workaround action that did this, nice to have native support 🚀